install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -58,6 +58,8 @@ spec:
 {{ toYaml .Values.affinity | indent 8 }}
 {{- end }}
 {{- if .Values.imagePullSecrets }}
+      nodeSelector:
```

Comment:
should this `nodeSelector` be independent with `imagePullSecrets` ?

Reply:
I agree it should.

Reply:
yes, I made a mistake and I have already fixed it, thanks!